feat:add feature to change password for profile as well as user

ChayanDass opened the pull request:
closes #523 


# password change operation

<img width="667" height="111" alt="image" src="https://github.com/user-attachments/assets/0d8ab300-e999-461a-ad26-b791340bb6bf" />

logs for user password update and own password 

<img width="723" height="228" alt="image" src="https://github.com/user-attachments/assets/e918c90d-7f70-43de-9c68-7e1b1186ce95" />

ChayanDass commented:
@bupd, hey, could you check this PR once?

bupd commented on harbor-cli:
please remove the binary files

bupd reviewed (changes requested):
Overalll LGTM, will give it a hands on review in today's CLI meeting.

bupd reviewed (commented):
@ChayanDass Thanks for you contributions. will review this next week. Please do update the screenshots on the PR description.

Vad1mo commented:
lint error, other wise loog good

bupd reviewed (approved):
lgtm

bupd reviewed (changes requested):
@ChayanDass please fix lint issue and make sure CI passes before requesting a review.

ChayanDass commented:
Hey @bupd, sorry for the earlier review request — I accidentally requested before checking the latest messages. I’ve now fixed the linting issues and updated the docs. 

Maybe I missed, but I'm  not able to find any instructions about the docs at `contributing.md`
